src/main/java/com/google/sps/data/Video.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+import java.util.List;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+
+public class Video {
```

Comment:
`Video` makes it seem like this class represent what a video. It is more representing of something like `VideoCommentsAnalysis` or something along those lines.